cmdeploy/src/cmdeploy/remote/rdns.py

```diff
@@ -34,7 +35,10 @@ def perform_initial_checks(mail_domain):
 
     # parse out sts-id if exists, example: "v=STSv1; id=2090123"
     parts = query_dns("TXT", f"_mta-sts.{mail_domain}").split("id=")
-    res["sts_id"] = parts[1].rstrip('"') if len(parts) == 2 else ""
+    if len(parts) == 2 and len(parts[1]) > 1:
```

Comment:
this could use a comment

Reply:
there is a much more elegant fix now <.<